src/NServiceBus.Gateway.PersistenceTests/When_checking_unknown_message.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace NServiceBus.Gateway.PersistenceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NUnit.Framework;
+
+    class When_checking_unknown_message : NServiceBusGatewayDeduplicationStorageTest
```

Comment:
Same question why not public?

Reply:
🤷‍♂ didn't see a need for it to be public. No strong opinion though.